Scripts/Python/grsnWallPython.py

```diff
@@ -137,22 +137,22 @@
 
 EscapeArtist = ptAttribActivator(67, "Escape Artist Catchall")
 
-MaleSuit = [
-    '03_MLHand_Suit',
-    '03_MRHand_Suit',
-    '03_MHAcc_SuitHelmet',
-    '03_MLFoot_Suit',
-    '03_MRFoot_Suit',
-    '03_MLegs_Suit',
-    '03_MTorso_Suit']
-FemaleSuit = [
-    '03_FLHand_Suit',
-    '03_FRHand_Suit',
-    '03_FHair_SuitHelmet',
-    '03_FLFoot_Suit',
-    '03_FRFoot_Suit',
-    '03_FLegs_Suit',
-    '03_FTorso_Suit']
+MaleSuit = {
+    0:'03_MLegs_Suit',
+    1:'03_MTorso_Suit',
+    2:'03_MLHand_Suit',
+    3:'03_MRHand_Suit',
+    5:'03_MHAcc_SuitHelmet',
+    6:'03_MLFoot_Suit',
+    7:'03_MRFoot_Suit'}
+FemaleSuit = {
+    0:'03_FLegs_Suit',
+    1:'03_FTorso_Suit',
+    2:'03_FLHand_Suit',
+    3:'03_FRHand_Suit',
+    5:'03_FHair_SuitHelmet',
+    6:'03_FLFoot_Suit',
+    7:'03_FRFoot_Suit'}
```

Comment:
Why not make these tuples? The keys are just sequential integers, which you get for free with tuples. You're then able to avoid performance penalties for looking these up multiple times in the later code.

Reply:
they are not sequential, 4 is missing

Reply:
Use a placeholder, like `None` for index 4 in that case. You can skip falsey values in a loop by doing `for suitItem in filter(None, FemaleSuit):`.